src/DefaultBuilder/src/ConfigureWebHostBuilder.cs

```diff
@@ -66,14 +65,13 @@ public IWebHostBuilder ConfigureServices(Action<IServiceCollection> configureSer
         /// <inheritdoc />
         public string? GetSetting(string key)
         {
-            _settings.TryGetValue(key, out var value);
-            return value;
+            return _configuration[key];
```

Comment:
It seems questionable to just mirror settings to config. HostBuilder doesn't do this, does it? Can we do something more limited?

Reply:
My bad. I was confusing settings and properties.